datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -43,11 +51,33 @@ class WinWMICheck(AgentCheck):
     """
 
     def __init__(self, *args, **kwargs):  # To support optional agentConfig
+        # type: (List[Any], Dict[str, Any]) -> None
```

Comment:
According to the [Python 2 cheatsheet](https://mypy.readthedocs.io/en/stable/cheat_sheet.html), this should be:

```suggestion
        # type: (*Any, **Any) -> None
```